lib/Sema/CodeSynthesis.cpp

```diff
@@ -1723,12 +1723,52 @@ bool swift::hasLetStoredPropertyWithInitialValue(NominalTypeDecl *nominal) {
   });
 }
 
+/// Determine whether a synth
```

Comment:
Nit: incomplete comment

Reply:
Thank you, fix incoming with another related PR

Reply:
(Comment completed in https://github.com/swiftlang/swift/pull/81450)